README.md

```diff
@@ -112,7 +112,16 @@ Welcome to contribute to Gluten project! See [contributing guide](CONTRIBUTING.m
 
 ## 4.1 Community
 
-You can join a Wechat group (Chinese) or a Spark channel in Velox Slack group (English) for community communication. Contact us if you want.
+Glunten joined Apache incubator as of March'24. 
+### Mailing lists
+For any technical questions, please email to dev@gluten.apache.org
+
+
+### Wechat group
+We also have a Wechat group (Chinese) which maybe more firendly for PRC customers. As there's a limitation of wechat group of 200 peoples, please contat with Kyligence assistent to be invited to the group. 
```

Comment:
please put the email as the contact: weitingchen@apache.org and zhangzc@apache.org